src/main/java/uk/gov/hmcts/dts/fact/services/admin/list/AdminAreasOfLawService.java

```diff
@@ -53,6 +56,16 @@ public AreaOfLaw createAreaOfLaw(final AreaOfLaw areaOfLaw) {
         return new AreaOfLaw(areasOfLawRepository.save(createNewAreaOfLawEntityFromModel(areaOfLaw)));
     }
 
+    public void deleteAreaOfLaw(final Integer areaOfLawId) {
+        try {
```

Comment:
Should we check the CourtAreaOfLaw, ServiceArea and CourtLocalAuthorityAreaOfLaw for usages of the AoL? CourtAreaOfLaw doesn't cascade deletes at present, which causes a FK constraint violation and an exception but that could change in the future.

Reply:
Yes definitely. Could change very easily. Like you said in dev stand-up, if we change it to cascade all, it will cause issues. 

I think the above comment about having a method, raising a custom exception etc would cover this too!